docs/topics/migration-best-practices.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/getting-started-guide/master.adoc
 
+:_content-type: REFERENCE
 [id="migration-best-practices_{context}"]
 = Best practices
 
```

Comment:
@stoobie @sbeskin-redhat I'd call this either a concept module or possibly a procedure module -- it describes our recommendations for managing a project, so I don't think it's a reference module. Reference modules are usually lists of commands or field values; this is a recommended procedure.

Reply:
@RichardHoch  I don't think it can be called a procedure. A procedure contains precise steps, and this module doesn't. I thought it was closer to reference than to a concept, for lack of a better-suited tag. But if you feel strongly about it, I can change it to concept.

Reply:
@sbeskin-redhat Please change this to a concept.